index.js

```diff
@@ -0,0 +1,51 @@
+const GIPHY_URL = 'https://api.giphy.com/v1/stickers/search?'; 
+const API_KEY = 'api_key=E26iZ0BIEszi7RA4GeEmXzJPRpkSeQ6m';
+const SEARCH_PARAM = '&q=';
+const AMOUNT = '&limit=50';
+
+let numberElements = 5;
+let userWord;
```

Comment:
`let userWord,
 url,
 json;`

Reply:
Overall it's bad idea to make such global variables. I see no reason for it at the moment